### DISCUSSION TITLE
Difference between IPDF and manual plotting of orientation angle

### DISCUSSION THREAD

**Discussion by aditya-shukl** [mtex-toolbox/mtex#1920] [Ask Anything]
![ipf plot](https://github.com/mtex-toolbox/mtex/assets/102997857/fa3dd87f-d7bc-4b28-9a30-d0449e1c1287)
Hi MTEX users and developers

I am seeking some advice for my IPF plotting to identify any texture.

ori2 is the list of orientations. colors is the ipf colorcoding. I give the command 

plotIPDF(ori2,colors,vector3d.Y,'MarkerSize',10.4)

This generates the figure called ipf plot attached . This tells me that along the y axis I see nearly all crystal directions.

But when I give the command 

h1=Miller ({1,1,1},cs)
cond = angle(yvector,ori2.*h1,'antipodal')./degree;
histogram(cond,10)

I get the histogram hist attached which suggests that most of the 111 crystal directions are not parallel to the yvector as the angles are so high. 

Why do I get different information. Am I interpreting something wrong here.

I want to understand if my sample is textured in this direction or not. 

Thanks for your help

Best
Adi
![hist](https://github.com/mtex-toolbox/mtex/assets/102997857/aae64638-5c6e-4c79-84f7-19c7399c4e94)

**Comment by kilir** [comment]
Hi Adi,
two thoughts: a) in an IPF you plot crystal directions, so in comparison ` angle(inv(ori) * yvector, h1, 'antipodal')` looks more comparable b) I'd expect for a uniform distribution to find more highly populated bins further away from 0 since you are on a sphere.

If you want to know where you have a non-uniform testure, you may try to contour your plot appropriatly which makes a visual inspection easier, and potentially compare against a uniform distribution.

Hope that helps.

Cheers,
Rüdiger
